hera_pspec/pspecdata.py

```diff
@@ -13,6 +13,8 @@
 import ast
 import glob
 
+SPEED_OF_LIGHT = 299792458.
```

Comment:
we have a `conversions.utils` object specifically for these values

Reply:
I've removed explicit references to speed of light. New implementation allows for arbitrary "clean" windows to be specified in which foregrounds are down-weighted.

Reply:
okay but, then can we get rid of this line then?

Reply:
seems like this line is still here. let's get rid of this if we arent' using it